sdcm/tester.py

```diff
@@ -395,11 +395,15 @@ def init_argus_run(self):
                     job_url=get_job_url(),
                     started_by=get_username(),
                     commit_id=get_git_commit_id(),
-                    runner_public_ip=get_sct_runner_ip(),
-                    runner_private_ip=get_my_ip(),
                     sct_config=self.params,
                 )
                 self.log.info("Initialized Argus TestRun with test id %s", self.test_config.argus_client().run_id)
+            self.test_config.argus_client().set_sct_runner(
+                public_ip=get_sct_runner_ip(),
+                private_ip=get_my_ip(),
+                region="undefined_region",
```

Comment:
Is it impossible to get runner region?

Reply:
ok

Reply:
> According to discussion with @k0machi we are not showing region of sct runner in Agrus now. Will create task to determine an post runner region with low priority. 

Runner is always in the first region from the test configuration.